ChangeLog.d/psa-openless.txt

```diff
@@ -0,0 +1,19 @@
+Features
+   * Use and access persistent keys directly from their key identifiers. As an
+     upgrade of the MbedTLS PSA Cryptography implementation from version 1.0
+     beta to the released version 1.0.0 it is now not necessary to open a
+     persistent key to use it. Opening a persistent key is still supported and
+     even not yet deprecated but will be deprecated in the coming months.
+
+Bugfix
+
+   * The build system now checks that MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER
+     and MBEDTLS_USE_PSA_CRYPTO are not enabled simultaneously as those two
+     configuration options are incompatible.
+   * psa_set_key_id() no longer resets the key location when enforcing the
+     key lifetime to be PSA_KEY_LIFETIME_PERSISTENT.
+   * Attempting to create a volatile key with a non-zero key identifier now
+     fails. Previously the key identifier was just ignored when creating a
+     volatile key.
+   * Attempting to register a key in a secure element or to create a persistent
+     key with a key identifier in the vendor range now fails.
```

Comment:
This looks like it's saying “Attempting to register a key in a secure element now fails”. Suggestion:

> Attempting to create or register a key with a key identifier in the vendor range now fails.